Source/FieldSolver/FiniteDifferenceSolver/MacroscopicProperties/MacroscopicProperties.H

```diff
@@ -30,6 +62,89 @@ private:
      amrex::Real m_epsilon;
      /** Permeability, mu, of the medium */
      amrex::Real m_mu;
+     /** Multifab for m_sigma */
+     std::unique_ptr<amrex::MultiFab> m_sigma_mf;
+     /** Multifab for m_epsilon */
+     std::unique_ptr<amrex::MultiFab> m_eps_mf;
+     /** Multifab for m_mu */
+     std::unique_ptr<amrex::MultiFab> m_mu_mf;
+     /** Stores initialization type for conductivity : constant or parser */
+     std::string m_sigma_s;
+     /** Stores initialization type for permittivity : constant or parser */
+     std::string m_epsilon_s;
+     /** Stores initialization type for permeability : constant or parser */
+     std::string m_mu_s;
+
+     /** string for storing parser function */
+     std::string m_str_sigma_function;
+     std::string m_str_epsilon_function;
+     std::string m_str_mu_function;
+     /** Parser Wrappers */
+     std::unique_ptr<ParserWrapper<3> > m_sigma_parser;
+     std::unique_ptr<ParserWrapper<3> > m_epsilon_parser;
+     std::unique_ptr<ParserWrapper<3> > m_mu_parser;
+};
+
+/**
+ * \brief
+ * This struct contains only static functions to compute the co-efficients for the
+ * Lax-Wendroff scheme of macroscopic Maxwells equations using
+ * macroscopic properties, namely, conductivity (sigma), permittivity (epsilon),
+ * and permeability (mu).
+ */
```

Comment:
Right now this is only used for macroscopic evolve E and doesn't require mu.  I don't think this stuff will ever apply to macroscopic evolve B since there that uses the dual-update predictor/corrector, and does not contain sigma that needs to be integrated to 2nd order in one function call.

Reply:
Thank you @ajnonaka. Yes, mu is not used to determine alpha and beta coefficients and mu is directly used in E-update as beta/mu. I have removed mu from here.
That is right. EvolveB is the same for vacuum and macroscopic.

Reply:
I added a comment to clarify that even though mu is not used in the co-efficients defined in the struct, it is used in E-update